.changeset/wide-trains-camp.md

```
@@ -0,0 +1,5 @@
+---
+"ensapi": minor
+---
+
+Adds a `canonical?: boolean` filter to the where filter in Queyr.domains and Account.domains. When specified, the resulting set of Domains is composed exclusively of Canonical Domains.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)